src/pages/MemberProfilePage.tsx

```diff
@@ -135,27 +117,34 @@ const Widgets: React.FC<{ member: Member }> = ({ member }) => {
           w="full"
           align="stretch"
         >
-          {member.leetcode && (
+          {member.leetcode && leetcodeStats && (
```

Comment:
Might be able to simplify this to just:
```suggestion
          {leetcodeStats && (
```
Since Leetcode stats shouldn't exist for members who haven't linked Leetcode.